From 2cf89e93be1fb80c71d0eeb041dc3a37590280fa Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 11 Sep 2013 00:03:48 +0200
Subject: [PATCH 14/41] Set TABSIZE globally

It's used outside ifdef'd parts and defined in ifdef'd parts redundantly.
---
 font.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/font.c b/font.c
index ba4c508..5949bec 100644
--- a/font.c
+++ b/font.c
@@ -32,6 +32,9 @@
 #include <string.h>
 #include "font.h"
 #include "koules.h"
+
+#define TABSIZE  64		/* size of a tab, in pixels */
+
 /*following routines are ripped from vgagl library */
 /* We use the 32-bit to 64-bit multiply and 64-bit to 32-bit divide of the */
 /* 386 (which gcc doesn't know well enough) to efficiently perform integer */
@@ -292,7 +295,6 @@ gl_wide_line (int x1, int y1, int x2, int y2, int c)
 
 
 
-#define TABSIZE  64		/* size of a tab, in pixels */
 #define vga_drawline(x1,y1,x2,y2)\
 	gl_wide_line(x1,y1,x2,y2,textcolor)
 #define vga_drawhline(x1,y1,x2,y2)\
@@ -406,7 +408,6 @@ SSetPixel (int x1, int y1, int c)
   lines[nlines].y2 = y1 + 1;
   nlines++;
 }
-#define TABSIZE  64		/* size of a tab, in pixels */
 #undef vga_drawline
 #undef vga_drawhline
 #undef vga_drawpixel
-- 
2.5.0

